libraries/bot-schema/src/main/java/com/microsoft/bot/schema/teams/MeetingDetailsBase.java

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.bot.schema.teams;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Specific details of a Teams meeting.
+ */
+public class MeetingDetailsBase {
+    @JsonProperty(value = "id")
+    private String id;
+
+    @JsonProperty(value = "joinUrl")
+    private String joinUrl;
+
+    @JsonProperty(value = "title")
+    private String title;

```

Comment:
Is Java case-insensitive when deserializing JSON? .NET is case-insensitive, but in JS, I had to [do some conversions](https://github.com/microsoft/botbuilder-js/blob/main/libraries/botbuilder/src/teamsActivityHandler.ts#L1038), since the all of the object property keys are TitleCased instead of camelCased (confusingly, ONLY FOR EVENTS--JSON payload for getMeetingInfo is camelCased)

Reply:
There is an option for this.  I enabled it.